### PR TITLE
Terraform config files

### LINKED ISSUE #2: Add setup script to integrate with tdp-getting-started
We need a new script to seamlessly integrate with `tdp-getting-started` as we cannot extend the `setup.sh` script there with an Alliage dependency.

### PR DESCRIPTION
**Pull request by Nuttymoon**
#### Which issue(s) this PR fixes:

Fixes #2 

#### Additional comments:

This PR adds the initial Terraform config files to the repo. All containers configuration are located in `local.auto.tfvars`.

#### Acknowledgment

To make clear that you license your contribution under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0) and that you give permission to [SAS Adaltas](https://www.adaltas.com/), you have to acknowledge this by using the following check-box.

- [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

- [x] I hereby agree to grant [Adaltas](https://www.adaltas.io/) a copyright license to use my contributions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
+
+Launch a fully-featured virtual TDP Hadoop cluster with a single command _or_ customize the infrastructure.
+
+This is an alternative to [tdp-vagrant](https://github.com/TOSIT-IO/tdp-vagrant) which uses LXD containers instead of VMs.
+
+## Requirements
+
+- LXD (to operate Linux containers)
+- Terraform >= 0.13 (to automatize containers lifecycle managment)
+
```
Comment (gboutry) [inline README.md]: Ansible is missing in requirements

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (wdavidw) [inline LICENSE]: License is MIT, Copyright is Adaltas

**Review by wdavidw — CHANGES_REQUESTED** [review]

File: local.auto.tfvars [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  {
+    name = "tdp_edge"
+    limits = {
+      cpu    = 2
+      memory = "2GB"
+    }
+  }
+]
+
+tdp_image = "images:centos/7"
```
Comment (leopaul36) [inline local.auto.tfvars]: Should we move to a more modern image / distro?

LXD fails on my side with `The image used by this instance requires a CGroupV1 host system` in Fedora 36

Romain has been working to make `tdp-collection-prerequisites` work with Rocky 8 so maybe we should use `images:rockylinux/8`?

Comment (wdavidw) [reply local.auto.tfvars]: Yes we should. For our coming customers, I don't wish to deploy with such a legacy OS as centos 7, we should validate Rocky as early as possible.

File: LICENSE (hunk 2 above)
Comment (Nuttymoon) [reply LICENSE]: Don't we wanna use the same license as all TDP projects (Apache 2.0)?

Comment (wdavidw) [reply LICENSE]: Nope, MIT is fine and compatible

File: local.auto.tfvars (hunk 3 above)
Comment (Nuttymoon) [reply local.auto.tfvars]: Working on this.

**Comment by Nuttymoon** [comment]
Changed license and moved to Rocky 8 image by default. Updated README accordingly.

**Pull request merged**